### ISSUE TITLE
Fps drop in godot 3.3.4

### ISSUE THREAD

**Issue by CHIESOFTWARE** [godotengine/godot#53587]
### Godot version

3.3.4

### System information

Ubuntu Studio 21.04, AMD Radeom , GLESS 3

### Issue description

Running my project on godot 3.3.4 the fps drops to about 20fps. In godot 3.3.3 this same project maintains a stable 60 fps all the time. 

The game in question is a 2D game, I started its development with godot 3.1 making prototypes, only with this latest version I have had problems with 2D fps. 

### Steps to reproduce

IN mi case just have to run a 2D project , the fps are less than older version of godot.

**Comment by lawnjelly** [comment]
Must be something very simple ...

If this list of commits is correct:
https://github.com/godotengine/godot/compare/3.3.3-stable...3.3.4-stable
There's no changes to rendering (bar a state change in multimesh).

**Comment by akien-mga** [comment]
Also make sure to try 3.3.3 again to confirm that performance is back to 60 FPS. The drop might be linked to a system upgrade and not Godot.

**Comment by CHIESOFTWARE** [comment]
Hello, sorry the delay,  I did no get notification of answer. I think the drop might be linked to a system upgrade and not Godot like @akien-mga said. 

I only have one project and so far I have not had problems with updates, my computer is updated frequently.

Days ago the fps stayed below 30 when I edited my project with Godot 3.4, now I test again and the project goes to 60fps as usual.

I have not changed anything in the project that affects the fps, the only thing that had changed was the version of godot.

I guess some system update affected the video card or something like that.
